web/packages/test/scripts/set-env.sh

```diff
@@ -82,19 +86,21 @@ export RANDAO_COMMIT_DELAY="${ETH_RANDAO_DELAY:-3}"
 export RANDAO_COMMIT_EXP="${ETH_RANDAO_EXP:-3}"
 
 export DEFAULT_FEE="${ETH_DEFAULT_FEE:-1}"
-export DEFAULT_REWARD="${ETH_DEFAULT_REWARD:-1}"
 
 export CREATE_CALL_INDEX="${ETH_CREATE_CALL_INDEX:-0x3500}"
-export DISPATCH_GAS="${ETH_DISPATCH_GAS:-500000}"
 
 export REGISTER_NATIVE_TOKEN_FEE="${ETH_REGISTER_NATIVE_TOKEN_FEE:-0}"
 export SEND_NATIVE_TOKEN_FEE="${ETH_SEND_NATIVE_TOKEN_FEE:-0}"
 
 ## Vault
-export BRIDGE_HUB_INITIAL_DEPOSIT="${ETH_BRIDGE_HUB_INITIAL_DEPOSIT:-1000}"
+export BRIDGE_HUB_INITIAL_DEPOSIT="${ETH_BRIDGE_HUB_INITIAL_DEPOSIT:-10000000000000000000}"
```

Comment:
Why the big increase? In what currency is this? DOT?

Reply:
It's Ether with the precision of 18, so it's 10 Ether.

Reply:
Cool! A comment stating that would be super helpful.

Reply:
https://github.com/Snowfork/snowbridge/pull/940/commits/aab0ec007f47982b3cf871e151341a94fd341cbc